### PR TITLE
[16.0] [MIG] pos_partner_firstname

### PR DESCRIPTION
**Pull request by victor-champonnois**
@robinkeunen @huguesdk @remytms @carmenbianca 

internal task : https://gestion.coopiteasy.be/web#id=10959&action=475&active_id=492&model=project.task&view_type=form&menu_id=

### CODE REVIEW

File: pos_partner_firstname/models/pos_session.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class POSSession(models.Model):
+    _inherit = "pos.session"
+
+    def _loader_params_res_partner(self):
+        res = super()._loader_params_res_partner()
+        res["search_params"]["fields"].append("firstname")
+        res["search_params"]["fields"].append("lastname")
+        res["search_params"]["fields"].append("is_company")
```
Comment (huguesdk) [inline pos_partner_firstname/models/pos_session.py]:
```suggestion
        res["search_params"]["fields"].extend(["firstname", "lastname", "is_company"])
```

**Review by huguesdk — CHANGES_REQUESTED** [review]

**Review by huguesdk — APPROVED** [review]

**Review by carmenbianca — APPROVED** [review]
LGTM. 699692a330921b8692968c347f9da03e2ee64692 (last commit) should be squashed into the commit before it.

**Pull request merged**